setup.cfg

```diff
@@ -0,0 +1,45 @@
+[metadata]
+author = SciTools
+author_email = scitools-iris-dev@googlegroups.com
+classifiers =
+    Development Status :: 1 - Planning
+    Intended Audience :: Science/Research
+    License :: OSI Approved :: BSD License
+    Natural Language :: English
+    Operating System :: OS Independent
+    Programming Language :: Python :: 3
+    Programming Language :: Python :: 3.6
+    Programming Language :: Python :: 3.7
+    Programming Language :: Python :: 3.8
+    Topic :: Scientific/Engineering :: Atmospheric Science
+description = Unstructured mesh library for Iris
+long_description = file: README.md
+long_description_content_type = text/markdown
+keywords =
+    iris
+    ugrid
+license = BSD 3-Clause License
+license_file = LICENSE
+name = iris_ugrid
+project_urls =
+    Code = https://github.com/SciTools-incubator/iris-ugrid
+    Issues = https://github.com/SciTools-incubator/iris-ugrid/issues
+url = https://github.com/SciTools-incubator/iris-ugrid
+version = attr: iris_ugrid.__version__
+
+[options]
+packages = find:
+python_requires =
+    >=3.6,<3.9
+setup_requires =
+    setuptools>=40.8.0
+zip_safe = False
+
+[tool:pytest]
+testpaths =
+    iris_ugrid/
+addopts =
+    -ra
+    -v
+    --doctest-modules
+doctest_optionflags = NORMALIZE_WHITESPACE ELLIPSIS NUMBER
```

Comment:
I think most software likes a new line at the end. This has been picked up in the GitHub diff anyway

Reply:
Another good spot thanks! Have added a newline now